Patches/FoodPoisoningChanceFix.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Harmony;
+using UnityEngine;
+
+namespace HungerRevamped {
+
+	/*
+	 * For some reason, food items have two cutoff values for food poisoning:
+	 *         x > 0.745: No   food poisoning chance
+	 * 0.745 > x > 0.200: Low  food poisoning chance
+	 * 0.200 > x        : High food poisoning chance
+	 *
+	 * When knowing how this system works, it ruins the randomness of the food poisoning system.
+	 * It's always just "check if below 20%, if yes, throw away".
+	 * A 74% condition food item should not have the same food poisoning chance as a 21% condition one.
+	 * So let's fix this:
+	 */
+
```

Comment:
The process for this follows similarly to the Cooking Skill fix. However, part of this feature is implemented in the `FoodPoisoningStart` patch that already existed for the delayed food poisoning feature.